relayer/pkg/chainlink/ocr2/config_digester.go

```diff
@@ -57,18 +57,12 @@ func (d offchainConfigDigester) ConfigDigest(cfg types.ContractConfig) (types.Co
 		oracles = append(oracles, signer, transmitter)
 	}
 
-	offchainConfig := EncodeBytes(cfg.OffchainConfig)
+	offchainConfig := starknet.EncodeBytes(cfg.OffchainConfig)
```

Comment:
Shouldn't this fn also be renamed as `DecodeBytes` considering the usual encoding/decoding process is:

```go
Encode(T) ([]byte, error)
Decode([]byte) (T, error)
```

Reply:
Also I'd consider renaming as `DecodeFelts` & `EncodeFelts` 🤔

Reply:
I'll take a look at this in a separate PR, but yeah I agree the naming isn't very clear